src/browser/components/SettingsPage.jsx

```diff
@@ -419,6 +440,48 @@ const SettingsPage = createReactClass({
         </Checkbox>);
     }
 
+    if (process.platform === 'darwin') {
+      options.push(
+        <FormGroup>
+          <Checkbox
+            inline={true}
+            key='bounceIcon'
+            id='inputBounceIcon'
+            ref='bounceIcon'
+            checked={this.state.notifications.bounceIcon}
+            onChange={this.handleBounceIcon}
+            style={{marginRight: '10px'}}
+          >{'Bounce the dock icon'}
+          </Checkbox>
+          <Radio
+            inline={true}
+            name='bounceIconType'
+            value='informational'
+            disabled={!this.state.notifications.bounceIcon}
+            defaultChecked={
+              !this.state.notifications.bounceIconType ||
+              this.state.notifications.bounceIconType === 'informational'
+            }
+            onChange={this.handleBounceIconType}
+          >{'once'}</Radio>
+          {' '}
+          <Radio
+            inline={true}
+            name='bounceIconType'
+            value='critical'
+            disabled={!this.state.notifications.bounceIcon}
+            defaultChecked={this.state.notifications.bounceIconType === 'critical'}
+            onChange={this.handleBounceIconType}
+          >{'until I open the app'}</Radio>
```

Comment:
@jasonblais Should we capitalize this sentence?

Reply:
Now, it'll appear as an option similar to `once`

![image](https://user-images.githubusercontent.com/13119842/32843013-a6453922-c9ec-11e7-8526-a99fec832ed8.png)